docs/faqs/contributor-faq.md

```diff
@@ -0,0 +1,250 @@
+Contributor FAQ
```

Comment:
this entire Contributor FAQ looks like a shorter (older?) version of the Developer FAQ. Was this supposed to be deleted?

Reply:
yes, we're creating three types of documentation focus:
1. user documentation
2. developer documentation
3. contributor documentation

Reply:
(thinking of creating..)